src/helpers/getDurationToExpiration.js

```diff
@@ -0,0 +1,11 @@
+export default expiryDate => {
```

Comment:
I will just say 'SENSEI IS A MATHEMATICIAN OOOO'. Damn!!! see logic. Weldone sir